cmd/config.go

```diff
@@ -86,6 +87,17 @@ type Config struct {
 	CheckpointSyncingProviders []string
 }
 
+// Viper doesn't work when you have environment variables. This is the
+// suggested workaround:
+// https://github.com/spf13/viper/issues/380
+func GetStringSliceWorkaround(flagName string) []string {
```

Comment:
I applied this thing pretty quickly, but I think we need to keep it generally so I'd leave it. If you don't have it then it literally can't process comma-separated values when you pass them as env vars.